.github/workflows/increment-patch-version/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Increment Patch Version 
+description: On a GA release already exists, the patch version should be incremented by 1.
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+          # Get package latest version
+          current_package_version=$(node -p "require('./package.json').version")
+          major_minor=$(echo "$current_package_version" | cut -d'.' -f1-2)
+
+          # !/-rc/ It excludes any lines containing the substring "-rc".
+          # !/refs\/tags\/v/ It excludes any lines containing the substring "refs/tags/v".
+          # !/beta/ It excludes any lines containing the substring "beta".
+          # !/dev/ It excludes any lines containing the substring "dev".
+          # !/cloud/ It excludes any lines containing the substring "cloud".
+          # /[0-9]+\.[0-9]+\.[0-9]+$/ It matches lines that end with a string in the format "x.y.z" where x, y, and z are one or more digits.
+          # $0 ~ /'"$major_minor"'/ It matches lines that contain the value of the major_minor variable.
+
+          latest_ga_version=$(git ls-remote --tags | awk '!/-rc/ && !/refs\/tags\/v/ && !/beta/ && !/dev/ && !/cloud/ && /[0-9]+\.[0-9]+\.[0-9]+$/ && $0 ~ /'"$major_minor"'/ {print $0}' | awk -F/ '{print $3}' | sort -rV | head -n 1)
+
```

Comment:
Instead of excluding, please include only the GA versions, see here: https://regex101.com/r/Zw8vsI/1

Reply:
Look like there is some issue with the regex, I added a real `git ls --tags`output to the regex snippet : 
https://regex101.com/r/Zw8vsI/2

Reply:
See https://regex101.com/r/Zw8vsI/3

Reply:
Look like there is some issue with the regex, I added a real `git ls --tags`output to the regex snippet : 
https://regex101.com/r/Zw8vsI/2

I updated the regex to fit the right git ls tags output, I updated the PR.
Can you check it out to see if this good enough? 